src/analysis/retail/hunter/survival/modules/talents/RuthlessMarauder.tsx

```diff
@@ -0,0 +1,69 @@
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import TALENTS from 'common/TALENTS/hunter';
+import Events, { DamageEvent } from 'parser/core/Events';
+import SPELLS from 'common/SPELLS/hunter';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+import {
+  RUTHLESS_MARAUDER_CDR,
+  RUTHLESS_MARAUDER_CDR_COOLDOWN_MS,
+} from 'analysis/retail/hunter/survival/constants';
+import HIT_TYPES from 'game/HIT_TYPES';
+
+/**
+ * https://www.wowhead.com/spell=385718/ruthless-marauder
+ *
+ * Fury of the Eagle now gains bonus critical strike chance against targets below 50% health,
+ * and Fury of the Eagle critical strikes reduce the cooldown of Wildfire Bomb and Flanking Strike by X sec.
+ *
+ * Warcraft log:
+ * https://www.warcraftlogs.com/reports/ByAFTgt94fYJ8vjQ#fight=7&type=damage-done&source=7&translate=true
+ */
+class RuthlessMarauder extends Analyzer {
```

Comment:
Something you can consider doing is also implementing the fact that Ruthless Marauder gives 15% crit flat to FotE always. This is a left-over effect blizzard has forgotten to remove from RM since 2022 😄 https://github.com/simulationcraft/simc/blob/thewarwithin/engine/class_modules/sc_hunter.cpp#L5940-L5942